.gitignore

```diff
@@ -158,3 +158,8 @@ sdk/cosmos/azure-cosmos/test/test_config.py
 
 # temporary folder to refresh SDK with cadl
```

Comment:
All new work should go under the sdk/ai/azure-ai-project folder. 

The sdk/ai/azure-ai-client folder is stale and will be deleted soon.

Reply:
Fixed, reflected the same change in azure-ai-projects.